manifests/common/plumgrid.pp

```diff
@@ -11,6 +14,16 @@
     owner   => root,
     group   => root,
     mode    => '0440',
-    content => 'nova ALL=(root) NOPASSWD: /opt/pg/bin/ifc_ctl_pp *\n',
+    content => "nova ALL=(root) NOPASSWD: /opt/pg/bin/ifc_ctl_pp *\n",
+  }
+
+  openstack::resources::firewall { 'Nova API': port => '8774', }
+  openstack::resources::firewall { 'Nova Metadata': port => '8775', }
+
+  class { '::nova::api':
```

Comment:
nova profile is managing number of things which PLUMgrid doesn't require. It just need nova-api for it's metadata services. I don't thing it would harm here since it's completely a plumgrid common class, won't going to effect any thing other.